src/components/dialogs/refund.vue

```diff
@@ -109,10 +110,12 @@ export default class RefundDialog extends Mixins(PaymentMixin, PaymentSessionMix
   /** Called when user clicks "Cancel this NR" button. */
   private async confirmRefund (): Promise<void> {
     this.loading = true
-    if (await this.patchNameRequestsByAction(NrAction.REFUND)) {
+    const data = await NamexServices.patchNameRequestsByAction(this.getNrId, NrAction.REFUND)
+    if (data) {
       this.loading = false
       await this.hideModal()
       this.setDisplayedComponent('Success')
+      await this.loadExistingNameRequest(data)
```

Comment:
This change needs to be verified (tested). It looks fine but I just think we need to see it in action in Dev or Test.

Reply:
like all the other stuff this was in test and tested and is now in prod, but the more testing the better for sure